Resources/Prototypes/Entities/Objects/Weapons/Guns/turrets.yml

```
@@ -73,52 +73,81 @@
-            - !type:SpawnEntitiesBehavior
-              spawn:
-                SheetSteel1:
-                  min: 3
-                  max: 5
     - type: Gun
       fireRate: 6
       selectedMode: FullAuto
       availableModes:
         - FullAuto
       soundGunshot: /Audio/Weapons/Guns/Gunshots/gun_sentry.ogg
+      currentAngle: 25 # Fires first shot at min accuracy.
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)